api-reference/30 Data Layer/PivotGridDataSource/3 Methods/state().md

```diff
@@ -111,3 +111,7 @@ The **PivotGridDataSource** state - fields configuration, filters, expanded head
     export default App;
 
 ---
+
+#include common-demobutton with {
+    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/PivotGrid/StatePersistence/"
```

Comment:
The State Persistence demo uses state(state) to set state only. The Standalone Field Chooser demo uses the state() itself. Did you place the StatePersistence demo inside state(state) on purpose?

Reply:
state() and state(state) are basically the same method, so I choose State Persistence because it demonstrates the functionality for which these methods were added.